Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:boron
+
+# Create app directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Install app dependencies
+COPY package.json /usr/src/app/
+RUN npm install
+
+COPY . /usr/src/app
+
+EXPOSE 8080
```

Comment:
I agree with Gideon, i think you need a conditional to use process.env if it's going to be hosted ona  server